astropy/cosmology/core.py

```diff
@@ -99,13 +99,13 @@ class FLRW(Cosmology):
         Neff. Usually this means you must provide three neutrino masses unless
         you are considering something like a sterile neutrino.
 
-    name : str
-        Optional name for this cosmological object.
-
-    Ob0 : float
+    Ob0 : float or None
```

Comment:
Sorry, I realize it's not that you're specifying a default value, and I maybe shouldn't have used the word "default".  What I meant was that the default in this case is "unspecified" or N/A.  It's still optional though (if you dont' want it to be optional don't give it a default argument).